spec/computer_player_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../lib/computer_player.rb'
+require_relative '../lib/player.rb'
+
+describe ComputerPlayer do
+
+  let(:ai_double) { double("ai_double") }
+  let(:computer_player) { ComputerPlayer.new( {name: "Computer Player", marker: "X", ai: ai_double} ) }
+
+  it "is a subclass of the Player class" do
+    expect(ComputerPlayer.superclass).to eq(Player)
+  end
+
+  describe "#new" do
+  
+    it "accepts as an argument a hash of attributes to set the instance's #name, #marker, and #ai" do
+      attributes = {
+        name: "1",
+        marker: "X",
+        ai: ai_double}
+
+      computer_player = ComputerPlayer.new(attributes)     
+
+      expect(computer_player.name).to eq("1")
+      expect(computer_player.marker).to eq("X")
+      expect(computer_player.ai).to eq(ai_double)
+    end
+
+  end
+
+  describe "#make_selection" do 
+  
+    it "calls ai.find_best_move and passes it the board, deciding_player, and other_player arguments" do
```

Comment:
This test doesn't actually assert that `make_selection` returned the desired move.

Can you update this test to make `ai_double` return a value and assert that `make_selection` returned that value?